packages/perspective-jupyter/package.json

```diff
@@ -19,15 +19,23 @@
         "bench": "npm-run-all bench:build bench:run",
         "bench:build": "echo \"No Benchmarks\"",
         "bench:run": "echo \"No Benchmarks\"",
+        "build": "npm-run-all build:*",
+        "build:base": "node build.js",
+        "build:lib": "babel src/js/ --source-maps --out-dir dist/esm/",
+        "build:labextension": "jupyter labextension build . --debug",
+        "build:nbextension": "webpack --mode=production -c src/config/webpack.config.js",
+        "clean": "npm-run-all clean:*",
+        "clean:dist": "rimraf dist",
+        "clean:lib": "rimraf lib",
         "clean:screenshots": "rimraf \"screenshots/**/*.@(failed|diff).png\"",
+        "clean:labextension": "rimraf ../../python/perspective/perspective/labextension",
+        "clean:nbextension": "rimraf ../../python/perspective/perspective/nbextension/static",
```

Comment:
These tasks should go in `python/perspective/package.json`, yes?

Reply:
Either, I think it's better here since this is the file that actually generates those assets. A little weird if you `yarn clean` from this folder and it doesn't actually clean the stuff you want it to clean